docs/protocol/getting_started.md

```diff
@@ -62,24 +62,10 @@ git add pipelines/myexp
 git commit -m 'adding myexp scaffold'
 ```
 
-### Popper check
-
-Run popper check:
-
-```bash
-popper check
-```
-
-> **NOTE:** By default, `popper check` runs all commands directly on 
-the host. We recommend running an isolated environment. In order to do 
-this, one can create a pipeline using the `--env` flag of the `popper 
-init` command. For example, `popper init <pipeline> --env=alpine-3.4` 
-runs a command inside an `alpine-3.4` container.
-
-Once a pipeline is checked, one can show the logs:
+### Run a pipeline
 
 ```bash
-ls -l pipelines/myexp/popper_logs
+popper run myexp
```

Comment:
This was fine, the only issues is that `popper check` is now named `popper run`.

Reply:
@ivotron I thought we would be implementing some different features in the popper check command. I think I misunderstood, I will make the required changes.

Reply:
Completed.